assets/queries/k8s/kubelet_not_managing_ip_tables/test/negative1.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```

For `kubelet` commands you need to use it's docker image
```suggestion
      image: joaodanielrufino/kubelet
```